build.sbt

```diff
@@ -258,7 +258,7 @@ lazy val V = new {
   def dap4j = "org.eclipse.lsp4j" % "org.eclipse.lsp4j.debug" % lsp4jV
   val coursierInterfaces = "1.0.4"
   val coursier = "2.0.16"
-  val ammonite = "2.3.8-124-2da846d2"
+  val ammonite = "2.4.0-9-0017ff97"
```

Comment:
I'm hesitant to bump this. For example if you are using Scala 2.13 and use a `Try` in your script for example it will throw and exception and not work. https://github.com/com-lihaoyi/Ammonite/issues/1182

Reply:
Och, I wasn't aware of that :/ Will revert it

Reply:
Removed!